build/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+HOST_DIR=$(cd $(dirname "$0")/../; pwd)
+CONTAINER_DIR="/opt/rook.github.io"
+
+if [ -n "$HOST_USER" ]; then
+  DOCKER_RUN="docker run -t --user $HOST_USER:$HOST_USER -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine"
+else
+  DOCKER_RUN="docker run -t -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine"
+fi
```

Comment:
this gives flexibility to the runner / host to specify the mappings if any.  on mac, not specifying a user, the `node_modules` dependencies will be installed with current user permissions, however on linux / jenkins build, the `node_modules` dependencies maintain the `root:root` permissions from how the container is run.  then wiping the folder on the jenkins box becomes and issue due to root permissions.  this gives flexibility to the runner to specify the `$HOST_USER` user id.